sql/core/src/main/scala/org/apache/spark/sql/execution/SQLExecution.scala

```diff
@@ -71,13 +73,21 @@ object SQLExecution {
       // streaming queries would give us call site like "run at <unknown>:0"
       val callSite = sc.getCallSite()
 
+      val desc = Option(sparkSession.sparkContext
+        .getLocalProperty(SparkContext.SPARK_JOB_DESCRIPTION)) match {
+        case value : Some[String] =>
```

Comment:
Nit: `case Some(value) =>` and then you don't need `.get`.
I think this should be indented another unit, or else move the `getLocalProperty` call into a `val` above. Then the Option-match-Some-None pattern is kind of superfluous. Just check for null.

Reply:
Done